Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -366,9 +366,9 @@ def pr_pull
     odeprecated "`brew pr-pull --workflow`", "`brew pr-pull --workflows=`" if args.workflow.presence
 
     workflows = if args.workflow.blank?
-      args.workflows.presence || ["tests.yml"]
-    else
       [args.workflow].compact.presence || ["tests.yml"]
+    else
+      args.workflows.presence || ["tests.yml"]
     end
```

Comment:
I think the current version is correct, isn't it?

If `workflow.blank?`, use `workflows` or default, otherwise use `workflow` or default.

Reply:
Yeah right, the error is somewhere else. The whole syntax is a little bit weird to read. I need to debug this.